client/index.css

```diff
@@ -153,6 +154,34 @@ body .input.error {
 .main-column p {
   margin-top: 1px;
 }
+td, th { 
+  padding: 1.9rem;
+}
+table {
+  background: #d3d3d3;
+  max-width: max-content;
+  margin-bottom: 20px;
+}
+.session-column {
```

Comment:
please ensure that any CSS that is specific to the status component is in the CSS of the component instead of the general CSS, this is required because we'll implement lazy loading to load the minimum JS and CSS required by each page (that is, once a page is requested, its code is downloaded, while right now we load everything in one call).